example-compositor/compositor/src/main.rs

```diff
@@ -21,6 +21,73 @@ use webrender::api::units::*;
 #[cfg(target_os = "windows")]
 use compositor_windows as compositor;
 
+#[cfg(not(target_os = "windows"))]
```

Comment:
this change doesn't seem relevant?

Reply:
It was relevant in that `cargo build` from the root directory is broken on non-windows platforms. In any case, I have submitted https://phabricator.services.mozilla.com/D52686 which only includes https://github.com/servo/webrender/pull/3796/commits/34d64d063f28f03ffa07e2110b046aeda4acb0c1.